src/rpc/mempool.cpp

```diff
@@ -41,7 +42,10 @@ static RPCHelpMan sendrawtransaction()
             {"hexstring", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The hex string of the raw transaction"},
             {"maxfeerate", RPCArg::Type::AMOUNT, RPCArg::Default{FormatMoney(DEFAULT_MAX_RAW_TX_FEE_RATE.GetFeePerK())},
              "Reject transactions whose fee rate is higher than the specified value, expressed in " + CURRENCY_UNIT +
-                 "/kvB.\nSet to 0 to accept any fee rate.\n"},
+                 "/kvB.\nSet to 0 to accept any fee rate."},
+            {"maxburnamount", RPCArg::Type::AMOUNT, RPCArg::Default{FormatMoney(0)},
```

Comment:
Probably should be an options object instead (can replace maxfeerate).

Reply:
Conceptually I agree. I don't have much practical knowledge of how the RPC interface is used in the wild but wouldn't this break existing code that sets a `maxfeerate`? I also see that there are a few other calls that pass multiple options as separate arguments like `convertopsbt` and `gettxoutsetinfo` so maybe there is a case for making a broader change outside of the scope of this PR. 

But, if in practice this would not meaningfully break existing use cases then I will change it to an options object.

Reply:
Marking as resolved unless there is disagreement.